File: week-20/day-3/backedapi/src/main/java/com/greenfoxacademy/backedapi/models/ArrayHandler.java
package com.greenfoxacademy.backedapi.models;

import org.springframework.stereotype.Component;

@Component
public class ArrayHandler {
    private String what;
    private int[] numbers;
    private int[] result;

    public String getWhat() {
        return what;
    }

    public void setWhat(String what) {
        this.what = what;
    }

    public int[] getNumbers() {
        return numbers;
    }

    public void setNumbers(int[] numbers) {
        this.numbers = numbers;
    }

    public int[] getResult() {
        return result;
    }

    public void setResult(int[] result) {
        this.result = result;
    }
}

File: week-25/day-3/RestAPI_Exercises/RestApiWebApplication/Models/ArrayHandler.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RestApiWebApplication.Models
{
    public class ArrayHandler
    {
        public string What { get; set; }
        public int[] Numbers { get; set; }
        public int Result { get; set; }

        public ArrayHandler()
        {
        }

        public ArrayHandler(int result)
        {
            Result = result;
        }

        public ArrayHandler(string what, int[] numbers, int result)
        {
            What = what;
            Numbers = numbers;
            Result = result;
        }
    }
}
